inc/settings.php

```diff
@@ -125,7 +125,7 @@ function siteorigin_north_settings_init(){
 				'sidebar_width' => array(
 					'label'             => __( 'Sidebar Width', 'siteorigin-north' ),
 					'type'              => 'measurement',
-					'live'              => true,
+					'live'              => false,
```

Comment:
Sorry for the late reply.

This problem isn't restricted to just `0`, it happens when the sidebar has a ridiculously low number  like maybe `50px` or `5em` or `4%`.

One of the reasons is the [40px padding](https://github.com/siteorigin/siteorigin-north/blob/develop/sass/layout/_content-sidebar.scss#L15) we have for the sidebar. And the content tries to fit itself in the remaining space.

As you mentioned using `overflow-x: hidden` could affect some widgets. Maybe just use a minimum width of `100px` or `150px`. What do you think?